src/openrct2/localisation/LanguagePack.cpp

```diff
@@ -86,6 +88,9 @@ class LanguagePack final : public ILanguagePack
         {
             Memory::Free(fileData);
             log_error("Unable to open %s: %s", path, ex.what());
+            auto uiContext = OpenRCT2::GetContext()->GetUiContext();
```

Comment:
I think it would be better to move this call to the `Context.cpp`, line 366 - the catch clause that is called when it fails to load *any* language. That would also unbreak the Language Pack test.

Reply:
Worked like a charm:)